### PR TITLE
Branch01

### PR DESCRIPTION
**Pull request by ramizkazak**
nice :)

### CODE REVIEW

**Comment by ercancivi** [comment]
nice move :)

**Review by AzadBatch21 — APPROVED** [review]
Well done!

**Review by asuman75 — APPROVED** [review]
nice job

File: configuration.properties [hunk 1]
```diff
@@ -4,4 +4,4 @@ userName3=User84
 userName4=User114
 password=Userpass123
 browser=chrome
-enviroment=http://qa.trycloud.net/index.php/login?clear=1
\ No newline at end of file
+enviroment=http://qa.trycloud.net/index.php/login?clear=1
```
Comment (ramizkazak) [inline configuration.properties]: nice joob

**Review by Olesea1986 — COMMENTED** [review]
We are doing perfect!!!

**Review by ramizkazak — COMMENTED** [review]
go go go

**Review by ercancivi — APPROVED** [review]
done

**Comment by avsank** [comment]
Looks amazing!!!

**Pull request merged**